docs/index.rst

```diff
@@ -40,11 +40,11 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 |                   Integration                    |  |SUPPVER|    | |AUTO|    [1]_ |
 +==================================================+===============+================+
-| :ref:`aiobotocore`                               | >= 0.2.3      | No             |
```

Comment:
It seems like we are "dropping" support for some integrations here, these should likely have release notes for each... ?